assets/stylesheets/_vendor.scss

```diff
@@ -5,15 +5,15 @@
 .gridicon {
 	fill: currentColor;
 
-	&.needs-offset g {
+	&.needs-offset use {
```

Comment:
If the `svg4everybody` polyfill is active, there won't be any `use` elements to style. The polyfill will remove them and replace with the SVG markup it downloads with XHR.

We need a style that targets both `g` and `use` elements.

Reply:
Excellent catch, thanks @jsnajdr !

I have pushed a fix for this using `:first-child`. Waiting for `calypso.live` before testing on IE11.

Reply:
The fix appears to be working, from what I can tell.